loops.js

```diff
@@ -33,6 +38,13 @@
 // Tip: een getal is oneven als je het door 2 deelt en er blijft nog iets over. Een getal is even als je het deelt door 2 en er blijft niets over..
 // ==========================================
 
+for (let plekkie = 0; plekkie < 16; plekkie++) {
+   console.log(plekkie)
+if (plekkie%2 == 0)
+   console.log("is even")
```

Comment:
Vergeet plekkie niet mee te loggen: console.log(plekkie + " is even")
idem bij regel 46.